.github/workflows/ci.yml

```diff
@@ -9,19 +9,23 @@ on:
       - "**"
 
 jobs:
-  build:
+  tests:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node: [ 12, 14, 16 ]
+
     steps:
     - name: Checkout
       uses: actions/checkout@v2
 
     - name: Setup Nodejs
       uses: actions/setup-node@v1
       with:
-        node-version: 12
+        node-version: ${{ matrix.node }}
 
     - name: Setup npm
-      run: npm i -g npm@5.8.0
+      run: npm i -g npm@8.1.2
```

Comment:
can we just specify the major version and let `npm` install any latest minor versions